techbuzz/src/main/resources/templates/users/resentVerification.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en"
+      xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:th="http://www.thymeleaf.org"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      layout:decorate="~{layout}">
+<head>
+    <title>resent verification template</title>
+</head>
+<body>
+<div layout:fragment="content">
+
+    <div class="col-md-4 offset-4">
+
+        <div class="card">
+            <div class="card-header text-center">
+                <h3>Resent verification email Form</h3>
+            </div>
+            <div class="card-body">.
+                <form id="resentVerificationForm" method="post" th:action="@{/resentVerification}" th:object="${resentEmail}">
+
```

Comment:
To display Flash error message you need to add the something like this:

```
<div class="alert alert-danger" role="alert"  th:if="${message}">
          <span th:text="${message}"></span>
          <button aria-label="Close" class="btn-close" data-bs-dismiss="alert" type="button"></button>
</div>
```

Reply:
thanks for clarifying my doubt . Done